hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileChecksum.java

```diff
@@ -220,7 +217,7 @@ public void testStripedAndReplicatedFileChecksum() throws Exception {
     FileChecksum replicatedFileChecksum = getFileChecksum(replicatedFile,
         10, false);
 
-    if (expectComparableStripedAndReplicatedFiles()) {
+    if (expectComparableStripedAndReplicatedFiles) {
```

Comment:
We could keep the methods which returns the `isCompositeCRC` flag. This will reduce the diff and make the implementation flexible and maintainable for future changes.